### ISSUE TITLE
Proposal: clog-obj - detach-from-dom

### ISSUE THREAD

**Issue by infinae** [rabbibotton/clog#397]
Currently elements can be removed from the DOM via `remove-from-dom` but this has the effect of also removing jquery data and bound events: [.remove()](https://api.jquery.com/remove/). I think it would be nice to also have a method to detach elements [.detach()](https://api.jquery.com/detach/) as part of the clog-obj interface. This would allow CLOG to support a memoised approach to rendering pages.
